turbinia/jobs/analysis/docker_logs.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Job to execute Docker analysis task."""
+from __future__ import unicode_literals
+
+from turbinia.workers import artifact
+
+from turbinia.evidence import Directory
+from turbinia.evidence import RawDisk
+from turbinia.evidence import GoogleCloudDisk
+from turbinia.evidence import GoogleCloudDiskRawEmbedded
+from turbinia.evidence import ExportedFileArtifact
+from turbinia.evidence import ReportText
+
+from turbinia.jobs import TurbiniaJob
+from turbinia.workers.analysis import wordpress
+
+class DockerLogExtractionJob(TurbiniaJob):
+  """Wordpress configuration extraction job."""
```

Comment:
Looks like a find and replace issue needs to happen here.

Reply:
Fixed.